### ISSUE TITLE
Unable to setup k8 website locally.

### ISSUE THREAD

**Issue by neutr0n420** [kubernetes/website#44223]
**This is a Feature Request**

**What would you like to be added**
- I am on the MacOS with an M1 chip, trying to set up the website locally, but after following the process, it is showing the blank screen. I am following the docs to set up the project locally. I am attaching the SS for the same,
<img width="1440" alt="Screenshot 2023-12-06 at 12 13 13 AM" src="https://github.com/kubernetes/website/assets/106504872/b4354d2b-cdc7-4437-b31f-383ad5b0ea5d">
<img width="1150" alt="Screenshot 2023-12-06 at 12 13 27 AM" src="https://github.com/kubernetes/website/assets/106504872/f030cedb-6d9a-4903-826c-af3bf024dc57">
<img width="1440" alt="Screenshot 2023-12-06 at 12 14 01 AM" src="https://github.com/kubernetes/website/assets/106504872/4f3a5eee-116f-49a9-b9f1-48709704a4f1">

Is there something that I am doing wrong? 
**Why is this needed**

**Comments**

**Comment by sftim** [comment]
/retitle Add support for offline website previewing

based on https://kubernetes.slack.com/archives/C09R23FHP/p1701797990349859?thread_ts=1701716259.355739&cid=C09R23FHP

**Comment by neutr0n420** [comment]
Hey @sftim, I am not able to see any preview of the website, as a result it will be really difficult to contribute, is there someone who has encountered this issue earlier and would like to assist me on this?

**Comment by Gauravpadam** [comment]
I'm sorry I'm late on the conversation, Have you tried 'make serve' @neutr0n420?

**Comment by sftim** [comment]
@neutr0n420 if someone adds this feature, you'll be able to contribute.

In the meantime, you could also try a different connection to the internet.

**Comment by sftim** [comment]
/triage accepted
/priority backlog

(it's valuable, but other things are even more valuable IMO)

**Comment by aryanbramhane1122** [comment]
Yep I have tried that @gauravpadam , it is giving me another error something like `Error: error building site: failed to acquire a build lock: open /Users/neutron│
/coding/open_source/website/.hugo_build.lock: too many open files              │
make: *** [serve] Error 1`

**Comment by sftim** [comment]
Help with this is welcome.

**Comment by sergeyshevch** [comment]
@sftim Looks like help-wanted label is missing here

**Comment by sftim** [comment]
> Looks like help-wanted label is missing here

Ah, no: not every issue qualifies for that particular label, even for issues where we'd welcome help.

**Comment by sergeyshevch** [comment]
Should be fixed in https://github.com/kubernetes/website/pull/45375

**Comment by vaibhav2107** [comment]
Is this issue fixed in #45375 ? If yes, then we can close this issue

**Comment by lmktfy** [comment]
Fixed in PR https://github.com/kubernetes/website/pull/51344

/close
